src/pages/Timeline.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from "@mui/material";
+import ComposePost from "../components/ComposePost";
+import { Stack } from "@mui/system";
+import PostList from "../components/Posts/PostList";
+
+const styles = {
+  root: {
+    width: "100%",
+  },
+  postListContainer: { padding: 2, width: "50%" },
+};
+
+const Timeline = () => {
+  return (
+    <Stack direction="row" sx={styles.root}>
+      <Box>
+        <ComposePost />
+      </Box>
+      <Box sx={styles.postListContainer}>
+        <PostList />
+      </Box>
+    </Stack>
```

Comment:
I don't think this is the layout we want. If we do it like this then the `CompostPost` component will show up to the left of the `PostList` component (on the same row). We want `ComposePost` to be above the `PostList`. I think it should look something like this:

```suggestion
    <Stack direction="row" sx={styles.root}>
      <Box sx={styles.postListContainer}>
        <ComposePost />
        <PostList>
      </Box>
    </Stack>
```